src/lib/dai.c

```diff
@@ -321,6 +348,7 @@ struct dai *dai_get(uint32_t type, uint32_t index, uint32_t flags)
 /* called from src/ipc/ipc3/handler.c */
 void dai_put(struct dai *dai)
 {
+	tr_info(&dai_tr, "DAI: dai_put1()");
 	int ret;
```

Comment:
All of this debug prints should be removed from the RFC PR that you will send to SOF community.